src/app/clusters/bindings/bindings.cpp

```diff
@@ -165,15 +158,75 @@ CHIP_ERROR BindingTableAccess::Write(const ConcreteDataAttributePath & path, Att
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR SaveWithDecodableBindingList(const DecodableBindingListType & bindingList, size_t newTableSize,
+                                        FabricIndex accessingFabricIndex, EndpointId localEndpoint)
+{
+    // The binding table can be at most 480 bytes, allocate on heap to reduce static stack size allocation
```

Comment:
> The binding table can be at most 480 bytes

This is going to need to become configurable; different products will want different binding table sizes....